src/client/client.cpp

```diff
@@ -140,6 +141,33 @@ Client::Client(
 	m_cache_save_interval = g_settings->getU16("server_map_save_interval");
 }
 
+void Client::migrateModStorage()
```

Comment:
If you don't mind I propose moving the `m_mod_storage_database = new ...` code here too and renaming the method accordingly

Reply:
IMO the constructor should initialize the object's state completely. If `m_mod_storage_database` is not initialized in the constructor, then the `Client` object will not be ready to have methods such as `getModStorageDatabase` called on it. On the other hand, `migrateModStorage` just transforms state; it doesn't initialize it.

Reply:
Eh, whatever.